llms/googleai/shared_test/shared_test.go

```diff
@@ -373,6 +374,23 @@ func testTools(t *testing.T, llm llms.Model) {
 				},
 			},
 		},
+		{
+			Type: "function",
```

Comment:
It would be very nice if the two functions/tools would be different - otherwise these two are identical and how will the model distinguish between their functionalities?